src/python/phenix_apps/apps/otsim/infrastructure.py

```diff
@@ -60,7 +60,20 @@
       'to_current':          {'type': 'analog-read', 'modbus': {'scaling': 2}},
       'to_active_power':     {'type': 'analog-read', 'modbus': {'scaling': 2}},
       'to_reactive_power':   {'type': 'analog-read', 'modbus': {'scaling': 2}},
-    }
+    },
+    'generator': {
+      'voltage':        {'type': 'analog-read',       'modbus': {'scaling': 2}},
```

Comment:
@sivasath shouldn't generator voltage be adjustable as well (`analog-read-write`)?